README.md

```diff
@@ -1,3 +1,25 @@
 # openHAB Infrastructure Project
 
-This project contains infrastructure configurations and functionalities that are used by openHAB Community projects.
\ No newline at end of file
+This project contains infrastructure configurations and functionalities that are used by openHAB Community projects.
+
+## TVision 360 installation
+
+#### Environment variables
+CICD requires configurable project id, and package url for deployment/publishing the jar. Local development uses a respoilite url to fetch our custom versions of openhab packages. 
+
+Add to your local what the `.gitpod.yml` has.
+
+```
+echo "MVN_REPO_URL=\${CI_SERVER_URL}/\${CI_PROJECT_ID}/packages/maven" >> ~/.bashrc
+```
+
+##### Local Respoilite Maven artifactory
+
+Install [Respoilite](https://reposilite.com/guide/about)
```

Comment:
No longer valid, moved to Gitea, but aiming to publish this via GitHub moving forward.